src/main/java/lotto/view/output/OutputView.java

```diff
@@ -0,0 +1,50 @@
+package lotto.view.output;
+
+import java.util.List;
+import lotto.model.lotto.lottoNumber.Lotto;
+import lotto.model.lotto.lottoNumber.Lottos;
+import lotto.model.lotto.winningResult.rank.Rank;
+import lotto.model.lotto.winningResult.WinningResults;
+
+public class OutputView {
+    public void outputExceptionMessage(String message) {
+        System.out.println(message);
+    }
+
+    public void outputIssuedLottos(Lottos lottos) {
+        outputLottoAmount(lottos);
+        for (Lotto lotto : lottos.lottos()) {
+            List<Integer> ascendingLotto = Sorter.sortAscending(lotto.getNumbers());
+            System.out.println(Messages.ISSUED_LOTTO(ascendingLotto));
+        }
+    }
+
+    private void outputLottoAmount(Lottos lottos) {
+        System.out.println(System.lineSeparator() +
+                String.format("%d개를 구매했습니다.", lottos.lottos().size()));
+    }
+
+    public void outputWinningResultStartLine() {
+        System.out.println(System.lineSeparator() + "당첨 통계");
+        System.out.println(System.lineSeparator() + "---");
+    }
+
+    public void outputWinningRanks(WinningResults winningResults) {
+        List<Rank> descendingRanks = Sorter.sortDescending(Rank.findRanksExceptFail());
+        for (Rank rank : descendingRanks) {
+            System.out.println(String.format(
+                    "%s (%s원) - %d개"
+                    , Messages.MATCHING_CONDITION(rank)
+                    , Messages.PRICE(rank.getPrice())
+                    , winningResults.findLottoAmountByRank(rank)
+            ));
+        }
+    }
+
+    public void outputEarningsRate(double earningsRate) {
+        System.out.println(String.format(
+                "총 수익률은 %.1f%%입니다."
+                , Math.round(earningsRate * 10) / 10.0
```

Comment:
수익률을 계산하는 것까지 서비스 레이어 혹은 도메인 객체가 관리하면 좋을 것 같습니다! 받은 값을 가지고 무언가 동작을 수행하는 것은 view의 책임이 아니라고 생각하는데, 어떻게 생각하시는지 궁금합니다!

Reply:
옴 MAth.round 이부분 말씀하시는걸까요?
이부분은 서비스층에서 계산한 수익률을 받아서 반올림하는 부분인데요, 반올림에 대한 기준은 출력로직에 들어간다고 생각해서 뷰에서 관리했습니다!
만약 이부분까지 서비스층에 배치하면, 출력방식이 "소수점 셋째자리에서 반올림해라" 로 바뀌면 서비스의 코드를 수정해야 하니까요..!
어떻게 생각하시나요??

Reply:
요 부분이 반올림 로직이었군요. 하지만 `반올림`의 개념이나 `Math` 라는 클래스의 용도, 본질까지 본다면 여전히 연산의 영역이라고 생각이 되네요.

다시 보니 채영 님은 `String.format( )`을 사용하셨는데 `printf`를 써보시면 어떨까요?
```suggestion
          System.out.printf("총 수익률은 %.1f%%입니다.", earningsRate);
```
이렇게 하면 따로 Math.rount( )를 사용하지 않고도 명시한 자릿수 아래에서 반올림을 자동으로 해줍니다!

Reply:
오모나 너무 좋은 생각이에요!! 제가 이런 쪽으로 많이 안 알아보고 아는 상식 안에서 했던 것 같아요..
저렇게 하니깐 훨씬 UI 로직으로 응집되는 것 같아요 ㅎㅎ 해답 주셔서 너무 감사합니다!! 수정해볼게요!!